add Titans' Stronghold scenario (#9)
qwrtln/Homm3BG-mission-book: sections/Titans stronghold.tex

\subsection*{\MakeUppercase{Scenario length}}

This scenario plays out over 16 rounds. (15 in impossible)

\subsection*{\MakeUppercase{Player setup}}

\textbf{Player count:} 1 - 6

\textbf{Starting Resources:} 30 \svg{gold}, 8 \svg{building_materials}, 2 \svg{valuables}

\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}

\textbf{Starting Units:} 
    A Pack of \svg{bronze} units of your choice.
    A Few \svg{bronze} units of your choice.

\textbf{Town Buildings:} nothing

\subsection*{\MakeUppercase{Map setup}}

Take the following Map tiles and set them up as shown in the scenario map layout: (P for number of player)

\textbf{(1*P) Starting Map tile (I)}
\begin{itemize}
    \item Starting Tiles of your chosen factions.
    \item Ignore all of their yellow line.
\end{itemize}

\textbf{(2*P) × Far Map tile (II-III)}

\textbf{(2*P) × Near Map tile (IV-V)}

\textbf{(1*P) + 1 × Center Map tile (VI-VII)}

If you don't have enough VI-VII tile you can use another for tile near to tile I.
The center of this tile is the Titans' Stronghold.

\columnbreak

\subsection*{\MakeUppercase{Win conditions}}

> \textbf{Win:}
> \begin{itemize}
> 	\item Kill all unit in the Titans' Stronghold (the center of tile VI-VII next to the tiles I)
> \end{itemize}

Reviewer: Maybe if there is only a single constraint, it doesn't need to be in `itemize`

```suggestion
\textbf{Win:} Kill all unit in the Titans' Stronghold (the center of tile VI-VII next to the tiles I)
```

\subsection*{\MakeUppercase{Timed events}}

\textbf{4$^{th}$, 8$^{th}$ and 12$^{th}$ Round:}
\begin{itemize}
	\item Remove black cubes from windmills, water wheels and mystical gardens.
\end{itemize}

\subsection*{\MakeUppercase{The story}}

> TODO \end{multicols*}

Reviewer 2: Please finish the story and move it above scenario length and below the source.

> \subsection*{\MakeUppercase{The story}}
> 
> TODO \end{multicols*}

Reviewer: Can be removed, right?

```suggestion
```

\newpage

\subsection*{\MakeUppercase{Additional rules}}

\begin{itemize}

> 	\item Remember : the center of tile VI-VII next to all I, is the Titans' Stronghold.

Reviewer: ```suggestion
	\item Remember: the center of tile VI-VII next to all I, is the Titans' Stronghold.
```

> 	\item nobody can enter the titans' Stronghold before all other VII battle are flagged by any player.

Reviewer: > all other VII battle
What does this mean? Should it be "all other VII fields" or something like that?

```suggestion
	\item Nobody can enter the Titans' Stronghold before all other VII battle are flagged by any player.
```

Author: Yes, this can be "all other VII fields"

	\item After defeating a level VII neutral army, instead of resolving field the player chooses an option three times from the following list (an option may be chosen multiple times):
	\begin{itemize}
		\item Another player (you choose) gains 5 gold
		\item Another player (you choose) gains 2 building materials
		\item Another player (you choose) gains 1 valuable
	\end{itemize}
	Then flag the VII with faction cube.
	(In solo play : no bonus!)
	\item Ignore all yellow line in tiles I.
	\item You can use your build token to give your resources to another player.
	\item Two player can use your build token for exchange artifacts and/or spells.

> 	\item Whenever a player visits an Obelisk, that player rolls one tresure dice and one resource dice, chooses one of the dice and resolves its outcome dice.

Reviewer: ```suggestion
	\item Whenever a player visits an Obelisk, that player rolls one tresure die and one resource die, chooses one of the dice and resolves its outcome.
```

> 	\item When all VII (not includ the titans' Stronghold) are flagged, randomly draw and shuffle the following (next page) number of neutral Unit cards from each of their corresponding decks to create a separate deck of neutral Units for the titans' Stronghold (The deck of titans' Stronghold is sometimes cut in two because otherwise certain scenarios are impossible to implement with the cards of certain extensions.)

Reviewer: ```suggestion
	\item When all VII (not including the Titans' Stronghold) are flagged, randomly draw and shuffle the following (see the next page) number of Neutral Unit cards from each of their corresponding decks to create a separate deck of Neutral Units for the Titans' Stronghold (the deck of Titans' Stronghold is sometimes cut in two because otherwise certain scenarios are impossible to implement with the cards from certain expansions).
```

> 	\item Any time a Hero enters the titans' Stronghold they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to beat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.

Reviewer: ```suggestion
	\item Any time a Hero enters the Titans' Stronghold they draw 5 cards from the Titans' Stronghold deck instead of drawing them from the Neutral Unit card decks. The units are placed on the Combat board (see page 29, “Neutral Unit Setup” in the Core Rulebook). Players attempt to beat the units they find at the Titans' Stronghold. Any Neutral Units defeated during Combat in the Titans' Stronghold are returned to their respective Neutral Unit decks instead of Titans' Stronghold deck. Any Neutral Units surviving Combat at the Titans' Stronghold are shuffled back into the Titans' Stronghold deck. If there are not enough Unit cards in this deck, draw as many Unit cards as there are and place them on the Combat board.
```

	\item Combat in the Titans' Stronghold now costs 1 MP to extend per Combat round, just like Combat against non-Azure tier units.

> 	\item Additionally, no player can:\linebreak
> 	a) Attack other Heroes.\linebreak
> 	b) Capture a Mine or Settlement that is already controlled.

Reviewer: ```suggestion
	\item Additionally, no player can:
	\begin{itemize}
	  \item Attack other Heroes.
	  \item Capture a Mine or Settlement that is already controlled.
	\end{itemize}
```

It looks like this without nesting
![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/1a8e1031-b861-4135-bed6-0d977ec35ccf)

\end{itemize}

\newpage

\hommtable[]{27}{
  \centering
  \medskip
  \textbf{Strength of Titans' Stronghold Armies}\\
  \bigskip

  \newcommand{\bronze}[0]{\svg[12]{bronze}}
  \newcommand{\silver}[0]{\svg[12]{silver}}
  \newcommand{\golden}[0]{\svg[12]{golden}}
  \newcommand{\azure}[0]{\svg[12]{azure}}

  \begin{tabularx}{\linewidth}{p{0.15\linewidth}XXXX} & \darkcell{Easy} & \darkcell{Normal} & \darkcell{Hard} & \darkcell{Impossible}\\
    \darkcell[1.4]{1 player} 
        & \lightcell[1.4]{3\bronze 2\silver 1\golden 1\azure}
        & \lightcell[1.4]{2\bronze 2\silver 2\golden 1\azure}
        & \lightcell[1.4]{2\bronze 2\silver 2\golden 2\azure}
        & \lightcell[1.4]{1\bronze 2\silver 2\golden 3\azure}\\
    \darkcell[1.4]{2 players} 
	    & \lightcell[1.4]{5\bronze 5\silver 3\golden 1\azure}
	    & \lightcell[1.4]{4\bronze 5\silver 3\golden 2\azure}
	    & \lightcell[1.4]{2\bronze 5\silver 5\golden 3\azure}
	    & \lightcell[1.4]{1\bronze 5\silver 7\golden 4\azure}\\

>     \darkcell[1.4]{3 players} 

Reviewer: Starting 3 players, all cells need to be `1.6` to fit three lines in there 

![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/e988b1ae-f989-4cff-a4d5-75ac61260ff4)

Or you can use similar pattern with brackets as in the Sentinels scenario (and describe in additional rules) to save one line

![image](https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/0e8de99e-4c0d-4c6a-938e-0d3e42219de1)

        	1\bronze 5\silver 7\golden 4\azure}\\
	\darkcell[1.4]{5 players} 
	  	& \lightcell[1.4]{6\bronze 6\silver 4\golden 1\azure \linebreak
	  		Then\linebreak 
	  		7\bronze 6\silver 3\golden 2\azure}
	  	& \lightcell[1.4]{5\bronze 6\silver 4\golden 2\azure \linebreak
	  		Then\linebreak 
	  		5\bronze 6\silver 4\golden 3\azure}
	  	& \lightcell[1.4]{3\bronze 6\silver 6\golden 4\azure \linebreak
	  		Then\linebreak 
	  		3\bronze 6\silver 6\golden 4\azure}
	  	& \lightcell[1.4]{1\bronze 6\silver 8\golden 6\azure \linebreak
	  		Then\linebreak 
	  		2\bronze 6\silver 8\golden 5\azure}\\
	\darkcell[1.4]{6 players} 
	  	& \lightcell[1.4]{8\bronze 8\silver 4\golden 1\azure \linebreak
	  		Then\linebreak 
	  		7\bronze 7\silver 5\golden 2\azure}
	  	& \lightcell[1.4]{6\bronze 8\silver 4\golden 3\azure \linebreak
	  		Then\linebreak 
	  		6\bronze 7\silver 5\golden 3\azure}
	  	& \lightcell[1.4]{3\bronze 8\silver 7\golden 5\azure \linebreak
	  		Then\linebreak 
	  		3\bronze 7\silver 8\golden 4\azure}
	  	& \lightcell[1.4]{3\bronze 7\silver 11\golden 6\azure \linebreak
	  		Then\linebreak 
	  		3\bronze 8\silver 10\golden 6\azure}\\
  \end{tabularx}
}

\vspace{3em}

> \center\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}

Reviewer 2: Rename the images, so that they correspond

> \center\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}

Reviewer 2: Please describe the images. Right now, they don't explain the number of players. There should be something like "X Player Tile Setup".

Reviewer: ```
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
  \captionof{figure}{1 player}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-2.png}
  \captionof{figure}{2 players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-3.png}
  \captionof{figure}{3 players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-4.png}
  \captionof{figure}{4 players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-5.png}
  \captionof{figure}{5 Players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-6.png}
  \captionof{figure}{6 players}
\end{minipage}
```

<img width="595" alt="Screenshot 2024-06-25 at 9 22 51" src="https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/237c13d8-ff27-4eb4-9f83-49861bb3dd59">

> \center\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
> \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-2.png}
> \center\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-3.png}
> \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-4.png}
> \center\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-5.png}
> \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-6.png}

Reviewer: Avoid using `\center` as it can cause issues
```suggestion
\begin{center}
\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-2.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-3.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-4.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-5.png}
\includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-6.png}
\end{center}
```

Reviewer: See my other comment about maps https://github.com/qwrtln/Homm3BG-mission-book/pull/9#discussion_r1652142140
